gradle.properties

```diff
@@ -66,8 +66,8 @@ httpclient.version=4.5.8
 httpcore-nio.version=4.4.11
 httpcore.version=4.4.11
 httpmime.version=4.5.8
-jackson.version=2.9.9
-jackson-databind.version=2.9.9.3
+jackson.version=2.9.10
+jackson-databind.version=2.9.10
```

Comment:
If the version numbers are the same again, we can think about dropping the extra property.

Reply:
I do not know - as there are many more similiar vulnerabilities found within last time i suspect more to come in the future. And the Jackson team seems to address some of these via micro-patches to jackson-databind only.

Therefore it seems to be reasonable to let booth version independent of each other as of now...